pkg/proxy/proxy_integration_test.go

```diff
@@ -415,6 +430,41 @@ func TestIntegrationGetLoadtest(t *testing.T) {
 		assert.NotEqual(t, apisLoadTestV1.LoadTestErrored, dat.Phase)
 		assert.Equal(t, false, dat.HasTestData)
 	})
+
+	t.Run("Get loadtest details from gRPC/REST gateway", func(t *testing.T) {
+		req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("http://localhost:%d/v2/load-test/%s", restPort, expectedLoadtestName), nil)
+		require.NoError(t, err, "Could not create GET request")
+
+		res, err := http.DefaultClient.Do(req)
+		require.NoError(t, err, "Could not send GET request")
+		require.Equal(t, http.StatusOK, res.StatusCode)
+
+		defer func() {
+			err := res.Body.Close()
+			assert.NoError(t, err)
+		}()
+
+		restBody, err = ioutil.ReadAll(res.Body)
```

Comment:
Is there an assertion regarding `restBody` missing?

Reply:
It can not be missing, it can only be empty and the next sub-test asserts if it matches expectations. But anyway I added extra validation to fail early if it is empty

Reply:
Sorry but where is the assertion for `restBody`? I can't find it.

Reply:
in the next subtest, that asserts contents of the body, tries to decode it, etc.

Reply:
And here is commit that adds it https://github.com/hellofresh/kangal/pull/68/commits/412e07252c525933718bdc5ab598bc21ef0d22b8

Reply:
Hold on ... you are share context between sub tests on the same level? Why have them as subsets at all then? This seems wrong to me.

Reply:
this is how it is implemented here, and this is how v1 API is being tested, I'm trying not to touch anything yet

Reply:
Let's create a ticket to clean this up later across the tests What do you think?

Reply:
Sounds good to me. Created one.